src/renderers/WebGLRenderer.js

```diff
@@ -77,6 +78,8 @@ function WebGLRenderer( parameters ) {
 	this.domElement = _canvas;
 	this.context = null;
 
+	this.debug = new WebGLRendererDebug();
```

Comment:
I think it may be too early to create a whole class for it. I would just do `this.debug  = { checkShaderErrors: true }` for now. And document the object in the `WebGLRenderer` page.

Reply:
done and done